### DISCUSSION TITLE
Button component when used as an anchor doesn't get disabled

### DISCUSSION THREAD

**Discussion by josamit** [twilio-labs/paste#604] [General] [answered]
Hi team,

I noticed an issue with Button component, which when used as an anchor 

```
                  <Button
                    as='a'
                    disabled
                    variant='secondary'
                    data-testid='some_test_id'
                    href='/some/href'
                  >
                    Edit
                  </Button>
```

it doesn't get properly disabled. Here is the screen capture of such a case :

![Jul-24-2020 12-11-08](https://user-images.githubusercontent.com/62208722/88376823-d761d680-cda6-11ea-8947-2419a36fb91f.gif)

**Comment by SiTaggart** [comment, marked as answer]
HI there @josamit, thanks for using Paste.

What we have here is unfortunately not an issue with the Paste component as such, but a limitation of the Web platform. In HTML you aren't allowed to disable a anchor, the HTML element doesn't support the `disabled` property. So it's something we would strongly advise against doing in your experiences.

In cases like these we would suggest coming with your design partner to a Design Systems Office hours so we can better understand what you're trying to do, and potentially offer alternative solutions.
